packages/bap-ida-python/bap-ida-python.1.0.0~alpha/files/generate-config.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Create a bap.cfg file for use in bap-ida-python based upon path to BAP
+# passed to it as input as first argument
```

Comment:
The opamish way to do this is to add a file to `files` and perform a substitution in it.